### ISSUE TITLE
Automate PR comment that warns that a locale is "frozen"

### ISSUE THREAD

**Issue by peterbe** [mdn/yari#3198]
If someone makes a PR on some `ru` or `pl` pages in mdn/translated-content, we should post a PR comment (automatically) that explains that this locale is not actively maintained (i.e. not unfrozen).

**Comment by schalkneethling** [comment]
@Rumyra This might also be something we can take on. Not sure how often this happens, though. Perhaps it is not needed anymore?

**Comment by Rumyra** [comment]
Yes I think this comes back to the unfrozen locales and whether we keep them around, or whether we (mindfully) archive them - in which case this isn't an issue. We can move it to decide 👍
